src/main/java/gov/nasa/pds/tools/validate/BundleManager.java

```diff
@@ -332,6 +291,66 @@ public static Target getLatestBundle() {
         return(children);
     }
 
+    private static ArrayList<Target> _getCollectionFilesWithSameLogicalIdentifier(List<Target> latestCollectionList, List<Target> otherCollectionList) {
```

Comment:
@qchaupds sorry to nitpick so much here, but the `_get*` method naming is introducing a new style that we have not been using throughout the code. if there is a good reason for it, I am all in. but i would prefer to we stick to what we have been doing until we are able to spend some significant time refactoring the code.

Reply:
The underscore has been removed:

{pds-dev3.jpl.nasa.gov}/home/qchau/sandbox/validate 150 % grep "getCollectionFilesWithSameLogicalIdentifier("  ./src/main/java/gov/nasa/pds/tools/validate/BundleManager.java
    private static ArrayList<Target> getCollectionFilesWithSameLogicalIdentifier(List<Target> latestCollectionList, List<Target> otherCollectionList) {
            ignoreCollectionList =